tests/function-controller/testsuite/runtimes/java.go

```diff
@@ -0,0 +1,137 @@
+package runtimes
+
+import (
+	"fmt"
+
+	serverlessv1alpha2 "github.com/kyma-project/kyma/components/function-controller/pkg/apis/serverless/v1alpha2"
+)
+
+const basicJavaRequirementsTpl = `
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.project.kyma.serverless</groupId>
+    <artifactId>hello-world</artifactId>
+    <version>0.0.1</version>
+    <packaging>jar</packaging>
+
+    <name>hello-world</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>javax.ws.rs</groupId>
+            <artifactId>javax.ws.rs-api</artifactId>
+            <version>2.0</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.project.kyma.serverless</groupId>
+            <artifactId>serverless-java-sdk</artifactId>
+            <version>0.0.1</version>
+            <scope>compile</scope>
+        </dependency>
+		%s
+    </dependencies>
+</project>
+`
+
+const additionalLib = `
+<!-- https://mvnrepository.com/artifact/com.fasterxml.jackson.dataformat/jackson-dataformat-csv -->
+<dependency>
+    <groupId>com.fasterxml.jackson.dataformat</groupId>
+    <artifactId>jackson-dataformat-csv</artifactId>
+    <version>2.14.1</version>
+</dependency>
+`
+
+var basicDeps = fmt.Sprintf(basicJavaRequirementsTpl, "")
+var updatedDeps = fmt.Sprintf(basicJavaRequirementsTpl, additionalLib)
+
+const basicHandler = `
```

Comment:
For me `return Response.ok("%s").build();` was hard to understand. This is template string. Can you change its name, to help understand this code?